oeps/oep-0021-proc-deprecation.rst

```diff
@@ -138,7 +138,9 @@ Analyze
 When proposing a removal, consider the following analysis:
 
 * Usage - Which users and services are currently using the code proposed for
-  removal on your own open edX instance?
+  removal on your own open edX instance?  Perform a quick search across the edX
+  codebase to gauge the level of impact and identify potential stakeholders.
+  https://github.com/search?q=org%3Aedx+sample&type=Code
```

Comment:
I suggest we add an additional step to the "Communicated" phase, which is to notify the stakeholders identified in this search. Seems kinda pedantic, but can't hurt.

Reply:
Good call -- I added some info to the `Communicated` section